src/ActionModal/components/ActionModal.vue

```diff
@@ -379,6 +379,10 @@ export default {
         body: `You have successfully completed a transaction.`
       })
     },
+    featureFlag: {
```

Comment:
nice!

Reply:
not really :/  
doesn't feel right to me.